chart/demoscenario.sh

```diff
@@ -0,0 +1,103 @@
+set -o errexit
+
+NAME=$1
+NAMESPACE=$2
+echo $EXTERNALNAME
+
+cat <<EOF_ | kubectl create -f -
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: ApplicationMapping
+metadata:
+  name: ${NAME}-app
+  namespace: ${NAMESPACE}
+  labels:
+    app: ${NAME}-app
+    chart: github-connector
+    release: ${NAME}
+EOF_
+
+echo `kubectl get serviceclasses -n rafal8 -o jsonpath="{.items}"`
```

Comment:
I'm not sure if rafal8 is the best demo namespace name

Reply:
True - removed.